src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/preferences/PreferencesPageObject.java

```diff
@@ -128,6 +131,28 @@ public void connectFacebook(String email, String password) {
     waitForElementByElement(facebookDisconnect);
   }
 
+  public void Use_advanced_recent_changes_checkbox() {
+    selectTab(PreferencesPageObject.tabNames.UNDER);
+    waitForElementClickableByElement(Use_advanced_recent_changes_checkbox);
+  Use_advanced_recent_changes_checkbox.click();
+    PageObjectLogging.log("Use_advanced_recent_changes_checkbox", "Use_advanced_recent_changes_checkbox clicked", true);
+  }
+
+  public void Get_advanced_recent_changes_checkbox_value() {
+// Verify that the Get_advanced_recent_changes_checkbox_value is checked
+    selectTab(PreferencesPageObject.tabNames.UNDER);
+    Assertion.assertFalse(Use_advanced_recent_changes_checkbox.getAttribute("checked") == null);
+     }
+
+  public void Set_advanced_recent_changes_checkbox_value_to_default_unchecked() {
+    selectTab(PreferencesPageObject.tabNames.UNDER);
+    if(Use_advanced_recent_changes_checkbox.getAttribute("checked") != null) // if Checked
+      Use_advanced_recent_changes_checkbox.click();
```

Comment:
for better readability consider wrapping that into {...} even if you executing one function